utils/utils.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from framework import config
+from framework.ssh_connection_manager import SSHConnectionManager
+from noobaa_sa.defaults import HEALTH
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+conf_root = config.ENV_DATA["config_root"]
+health = HEALTH
+base_cmd = f"sudo /usr/local/noobaa-core/bin/node {health}"
+unwanted_log = "2>/dev/null"
+
+
+# Function for gtting the noobaa health status
+def get_noobaa_health_status(config_root=None, **kwargs):
+    """
+    Noobaa Health Status
+
+    Args:
+        config_root (str): Path to config root
+
+        Supported update options via kwargs:
+        https_port (int): Get connection info
+        all_account_details (str): Get all bucket health info
+        all_bucket_details (str): Get all bucket health info
+
+    Example usage:
+        status = get_noobaa_health_status(
+            config_root=config_root,
+            https_port=https_port,
+            all_account_details=all_account_details
+            all_bucket_details=all_bucket_details
+        )
```

Comment:
Please add example values to `https_port`, `all_account_details` and `all_bucket_details` under "Example usage".  Reading this without being familiar with the CLI command, I'm still not really sure how to use the util function.